src/debugging/netcore/NetCoreDebugHelper.ts

```diff
@@ -228,6 +285,39 @@ export class NetCoreDebugHelper implements DebugHelper {
 
         return pathNormalize(result, platformOS);
     }
+
+    private async InstallDebuggerOnContainer(containerName: string): Promise<string> {
+        const debuggerPath: string = '/remote_debugger';
+        const installUnZip: string = '/bin/sh -c "apt-get update && apt-get install unzip"';
+        const installDebugger: string = `/bin/sh -c "curl -sSL https://aka.ms/getvsdbgsh | /bin/sh /dev/stdin -v latest -l ${debuggerPath}"`;

```

Comment:
The performance of this is going to be awful. Unlike in VS, we do not reuse containers in most cases, meaning we'll add several seconds (up to minutes) of launch time to every single container.

Even if we did reuse containers, it would still be seconds (up to minutes) per session.

Reply:
This is basically what VS does--I don't see a way around it if the user *really* wants to debug an arbitrary container (i.e. one that we didn't specifically create for the purposes of debugging).  I don't think the use case is repeated build/attach of a container, though, so the performance aspect is likely tolerable.

Reply:
If the intent is for debugging arbitrary containers, then I'm even more certain that we should not add this launch profile by default...

Reply:
The InstallDebugger command will skip if the latest version is already on the machine. But installing the unzip could be optimized in the future.

Reply:
Where is the logic to skip install if it's already there? Did I miss it?

Reply:
The script internally does it.

Reply:
Ah, yes. We should see how much time is also added by the `apt-get` call, including when `unzip` is already present.

Reply:
The installUnZip command takes 5.5 seconds if the unzip is not present. Second time it takes 2 seconds. The perf can be improved later. Not part of this iteration.

Reply:
I had a thought a while back about possibly pre-installing the debugger on new debug versions of the .NET Core Docker images, I wonder if we need to revisit that to save perf here...